packages/atomic/project.json

```diff
@@ -1 +1,264 @@
-{}
+{
+  "namedInputs": {
+    "tests": ["{projectRoot}/src/**/*.{spec.{ts,tsx},spec.{ts,tsx}.snap}"],
+    "e2e": ["{projectRoot}/cypress/**"],
+    "stories": ["{projectRoot}/src/**/*.{?(*([^.]).)stories.{tsx,css},mdx}"],
+    "source": [
+      "{projectRoot}/stencil.config.ts",
+      "{projectRoot}/collection-manifest.json",
+      "{projectRoot}/tailwind.config.js",
+      "{projectRoot}/svg.transform.js",
+      "{workspaceRoot}/tsconfig.json",
+      "{projectRoot}/tsconfig.json",
+      "{projectRoot}/src/**/*([^.]).!({{spec.{ts,tsx},spec.{ts,tsx}.snap},{?(*([^.]).)stories.{tsx,css},mdx}})",
+      "{projectRoot}/stencil-plugin/**"
+    ]
+  },
+  "targets": {
+    "clean": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "rimraf dist docs loader www storybook-static src/components/search/atomic-search-interface/lang src/generated",
+        "cwd": "packages/atomic"
```

Comment:
A big flaw with `nx:run-commands` is that the default `cwd` is the workspace root (`ui-kit`). Another flaw with the `cwd` is that it doesn't support variables nor `{projectRoot}`. We have to explicitly repeat `packages/atomic`.

It's ugly, but honestly not that big of a deal to copy and paste.